src/SFA.DAS.AssessorService.Database/Tables/Standards.sql

```diff
@@ -6,6 +6,7 @@
 	[Title] VARCHAR(1000) NOT NULL,
 	[Version] VARCHAR(20) NULL,
 	[Level] INT NOT NULL,
+	[CoronationEmblem] BIT NOT NULL DEFAULT 0,
```

Comment:
This column should also move to the end of the list to avoid the table rebuilding in the DacPac deployment

Reply:
Done